framework/src/mesh/MooseMesh.C

```diff
@@ -173,12 +178,30 @@ MooseMesh::MooseMesh(const InputParameters & parameters)
     _node_to_elem_map_built(false),
     _node_to_active_semilocal_elem_map_built(false),
     _patch_size(getParam<unsigned int>("patch_size")),
+    _ghosting_patch_size(isParamValid("ghosting_patch_size")
+                             ? getParam<unsigned int>("ghosting_patch_size")
+                             : 5 * _patch_size),
```

Comment:
And add a line:
```
_ghosting_patch_size(other_mesh._ghosting_patch_size),
```
into the copy constructor

Reply:
Thanks for figuring this out @andrsd. I will make these changes and update the PR.